src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/utils/OcrFieldExtractor.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.transformation.services.utils;
+
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.ocrvalidation.model.in.OcrDataField;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.transformation.model.in.ExceptionRecord;
+
+import java.util.List;
+
+public final class OcrFieldExtractor {
+
+    public static String get(ExceptionRecord er, String name) {
+        return get(er.ocrDataFields, name);
+    }
+
+    public static String get(List<OcrDataField> ocrFields, String name) {
```

Comment:
Why static? You can't mock static

Reply:
No need to mock it.
Not this is a util class.

Reply:
Now you have to know the logic provided by this utility class and prepare data for it, in order to test its client class, instead of just mocking its behaviour. Tests will unavoidably cover two classes at once.

Reply:
Ok, mocking in this case could require just as much effort. Never mind.